src/libcore/ptr/mod.rs

```diff
@@ -119,10 +119,13 @@ mod mut_ptr;
 ///
 /// Behavior is undefined if any of the following conditions are violated:
 ///
-/// * `to_drop` must be [valid] for reads.
+/// * `to_drop` must be [valid] for both reads and writes.
 ///
 /// * `to_drop` must be properly aligned.
 ///
+/// * The value `to_drop` points to must be valid for dropping, which may mean it must uphold
+///   additional invariants - this is type dependent.
```

Comment:
I'm not a native speaker, but I am wondering -- should this say "type-dependent"?

Reply:
As a half-native speaker, I think in English the rules governing dashes are quite loose, so both should be acceptable :) But sure, with one this is more readable.